### ISSUE TITLE
tabs v spaces

### ISSUE THREAD

**Issue by greggersaurus** [caseyokane/RaspberryProjects#2]
Pick one and be consistent.

**Comment by caseyokane** [comment]
Yeah, that was my bad. Was working on different terminals at separate times and forgot that the .vimrc settings weren't the same. Should be fixed now.
